src/golang/lib/models/views/artifact_node.go

```diff
@@ -0,0 +1,59 @@
+package views
+
+const ArtifactNodeViewSubQuery = `
```

Comment:
I'd like us to only keep struct definitions in `lib/models/views`. Any SQL queries should go under the proper `repos` directory. Something like this should belong in the `artifact` repo.

Reply:
Thanks! Would you mind reviewing #1165 and #1199 as well? Now that the stack is high and there will be lots of rebases, so I'd like to get all comments and address them all at once

Reply:
Yup reviewing rn

Reply:
Done :)